### DISCUSSION TITLE
🐞 Wrong URL of Audio in Webhook

### DISCUSSION THREAD

**Discussion by ricardojlrufino** [aldinokemal/go-whatsapp-web-multidevice#211] [General]
### What happened?

I receiving media type in audio url, like

 ```json
"audio": 
{
"media_path": 
"statics/media/1730293493-cd84c6a8-45aa-42bb-945f-0f576fb68aeb.ogg; codecs=opus",
"mime_type": 
"audio/ogg; codecs=opus",
"caption": 
""
}
```

And i cant download file using: /statics/media/1730293493-cd84c6a8-45aa-42bb-945f-0f576fb68aeb.ogg

### Reproduction Steps

```bash
1. Set webhhok
2. send ogg audio
```

### Screenshots or Additional Context

![image](https://github.com/user-attachments/assets/b0bc66ab-9407-495f-9ea0-59a29dc7fa33)

### Application Version

4.20.0

### Verification

- [X] I have checked for existing closed issues and discussions.

**Comment by aldinokemal** [comment]
Could you please share your OGG file here? I've checked on my side and everything seems fine, but I need your file to troubleshoot further

<img width="749" alt="image" src="https://github.com/user-attachments/assets/3b72c2ec-d7af-4d79-b412-264c41693dc7">

<img width="1143" alt="image" src="https://github.com/user-attachments/assets/39d9fcb4-acb5-48d4-854f-3a3f5547b88b">
